deeplake/core/tensor.py

```diff
@@ -1914,13 +2029,13 @@ def unload_vdb_index_cache(self):
             raise Exception(f"An error occurred while cleaning VDB Cache: {e}")
 
     def get_vdb_indexes(self) -> List[Dict[str, str]]:
-        if self.meta.htype != "embedding":
-            raise Exception(f"Only supported for embedding tensors.")
+        if self.meta.htype != "embedding" and self.meta.htype != "text":
+            raise Exception(f"Only supported for embedding and text tensors.")
```

Comment:
put these two line into separate validator function and call this function in all index related functions as above the same logics was remove

Reply:
Done. check_supported_tensor. This is places in create_vdb_index and delete_vdb_index. Update will not have this check as inverted index doesnt allow incremental index maintenance.